app/models/artist.rb

```diff
@@ -624,6 +612,18 @@ def search(params)
         q = q.includes(:tag).where("tags.name IS NULL OR tags.post_count <= 0").references(:tags)
       end
 
+      params[:order] ||= params.delete(:sort)
+      case params[:order]
+      when "name"
+        q = q.order("artists.name")
+      when "updated_at"
+        q = q.order("artists.updated_at desc")
+      when "post_count"
+        q = q.includes(:tag).order("tags.post_count desc nulls last").references(:tags)
```

Comment:
A pre-existing bug: this should be `order("tags.post_count desc nulls last, artists.name")`. Ordering by post count alone means the order is undefined when artists have the same post count.